parkings/api/enforcement/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.conf.urls import include, url
+
+from .active_parkings import ActiveParkingsView
+
+urls = [
+    url(r'^active-parkings/(?P<registration_number>.+)/',
```

Comment:
Convention on this project's (and on other HKI projects as well) API endpoint names is to use underscores instead of dashes and singular form instead of plural (so the name should be `active_parking`). Of course this isn't a public API so it doesn't matter that much, but I would still vote for consistency unless there are valid reasons to break it.

Reply:
About this endpoint in general, is there a reason why it works like `/active-parkings/<reg_num>/?time=<time>` instead of `/active-parkings/?reg_num=<reg_num>&time=<time>`?

The latter would be the "default" way of doing this in DRF, and I it would be more easily extended later if/when needed. For example we could easily add a way to query multiple registration numbers on one request (`/?reg_num=<reg_num>,<reg_num>,<reg_num>` for example), which is something we very well might need in the future.

Reply:
The reason to put the reg_num to path instead of query parameter was that it's mandatory.